src/components/inputs/rich-text-input/editor.js

```diff
@@ -192,200 +40,27 @@ const Editor = props => {
         return (
           <Constraints.Horizontal constraint={props.horizontalConstraint}>
             <Spacings.Stack scale="xs">
-              <Container
+              <RichTextBody
```

Comment:
I chose to separate out the "body", because the LocalizedInput will have a different look and functionality for it's expanding / collapsing buttons, etc.